sdist won't install on Anaconda, Python 3.5, Mac OS X

```
    Complete output from command //anaconda/envs/wasserstein/bin/python -u -c "import setuptools, tokenize;__file__='/private/tmp/pip-install-4x977wwt/minorminer/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /private/tmp/pip-record-qc1630p4/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_ext
    building 'minorminer' extension
    creating build
creating build/temp.macosx-10.6-x86_64-3.5
    creating build/temp.macosx-10.6-x86_64-3.5/python
    /usr/bin/clang -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I//anaconda/envs/wasserstein/include -arch x86_64 -I//anaconda/envs/wasserstein/include -arch x86_64 -I -I./include/ -I//anaconda/envs/wasserstein/include/python3.5m -c ./python/minorminer.cpp -o build/temp.macosx-10.6-x86_64-3.5/./python/minorminer.o -std=c++11 -Wextra -Wno-format-security -Ofast -fomit-frame-pointer -DNDEBUG -fno-rtti
    ./python/minorminer.cpp:567:10: fatal error: 'random' file not found
    #include <random>
             ^~~~~~~~
    1 error generated.
    error: command '/usr/bin/clang' failed with exit status 1
    
    ----------------------------------------
Command "//anaconda/envs/wasserstein/bin/python -u -c "import setuptools, tokenize;__file__='/private/tmp/pip-install-4x977wwt/minorminer/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /private/tmp/pip-record-qc1630p4/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /private/tmp/pip-install-4x977wwt/minorminer/
```

--- a/setup.py
+++ b/setup.py
@@ -2,6 +2,7 @@ from setuptools import setup, extension
 from setuptools.command.build_ext import build_ext
 import sys
 import os
+import platform
 
 cwd = os.path.abspath(os.path.dirname(__file__))
 if not os.path.exists(os.path.join(cwd, 'PKG-INFO')):
@@ -72,6 +73,8 @@ extensions = [Extension(
 
 if USE_CYTHON:
     extensions = cythonize(extensions)
+
+os.environ["MACOSX_DEPLOYMENT_TARGET"] = platform.mac_ver()[0]
 
 setup(
     name="minorminer",

